scripts/run-integration-tests.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-if [[ "$1" == "chrome" ]]; then
+if [[ "$1" == "chrome" && -n "$2" ]]; then
+  npm run test:integration -- --spec "$2" --browser chrome && echo "$2 tests passed" || bash scripts/report-failure.sh "$2 tests"
+elif [[ "$1" == "chrome" ]]; then
   npm run test:integration:chrome && echo "Integration tests passed" || bash scripts/report-failure.sh "Integration tests"
 elif [[ "$1" == "firefox" ]]; then
-  npm run test:integration -- --browser /usr/local/bin/firefox && echo "Integration tests passed" || bash scripts/report-failure.sh "Integration tests"
+  npm run test:integration -- --spec 'cypress/e2e/**/*.cy.ts' --browser /usr/local/bin/firefox && echo "Integration tests passed" || bash scripts/report-failure.sh "Integration tests"
```

Comment:
Do you still need `--spec 'cypress/e2e/**/*.cy.ts'` here if you already added it on cypress.config?